sunpy/instr/goes.py

```diff
@@ -1268,3 +1271,100 @@ def _calc_xraylum(flux, date=None):
     else:
         xraylum = 4 * np.pi * sun.constants.au.to("m")**2 * flux
     return xraylum
+
+def flareclass_to_flux(flareclass):
```

Comment:
Agreed. I wasn't aware of the other function. It does not seem to have been updated to use quantities so should probably go. The question now is do we move this into util or keep it here.